### PR TITLE
[15.0][MIG]sale_purchase_force_vendor: Migration to V15

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: sale_purchase_force_vendor/views/sale_order_view.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                expr="//field[@name='order_line']/form/group/group/field[@name='price_unit']"
+                position="before"
+            >
+            <field name="vendor_id_domain" invisible="1" />
+                <field
+                    name="vendor_id"
+                    domain="vendor_id_domain"
+                    options='{"no_create": True}'
+                />
+           </xpath>
```
Comment (sergio-teruel) [inline sale_purchase_force_vendor/views/sale_order_view.xml]:
```suggestion
            <xpath expr="//field[@name='order_line']/form/group" position="inside">
                <group string="Vendor" name="vendor">
                    <field name="vendor_id_domain" invisible="1" />
                    <field
                        name="vendor_id"
                        domain="vendor_id_domain"
                        attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
                        options='{"no_create": True}'
                    />
                </group>
            </xpath>
```

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
For better visibility and inheritance, Can you change this??

Thanks

File: sale_purchase_force_vendor/views/sale_order_view.xml (hunk 1 above)
Comment (bizzappdev) [reply sale_purchase_force_vendor/views/sale_order_view.xml]: Done @sergio-teruel

**Review by sergio-teruel — APPROVED** [review]
Thanks!!!!

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**